config/runtime.exs

```diff
@@ -0,0 +1,15 @@
+import Config
+
+config :cms, CMS.Cache,
+  conn_opts: [
+    host: System.get_env("REDIS_HOST", "127.0.0.1"),
+    port: 6379
+  ],
+  stats: false,
+  telemetry: false
+
+config :site, SiteWeb.Router,
+  cms_basic_auth: [
+    username: System.get_env("CMS_BASIC_AUTH_USERNAME", "username"),
```

Comment:
suggestion: can we make this a safer default? Since this is an open-source project, anyone not setting these variables (including us) will have an easily guessable password.

My vote would be to return `403 Unauthorized` for everything if the credentials are unset, but you could also generate a random username/password at startup and log it if you wanted.

Reply:
@paulswartz I just changed this so I can test it in dev-blue because I am still waiting for these to be set in my devops PR. In the second PR, it fails if these aren't set.

https://github.com/mbta/devops/pull/1434

https://github.com/mbta/dotcom/pull/1832/files#diff-e43c5cbf91db7a8062b6cb860cbf118296c1b4c7ee32fdcf702e54234ba38092